Dockerfile.dev

```diff
@@ -0,0 +1,18 @@
+# Base image for installing dependencies
+FROM node:20-alpine AS base
+
+RUN npm install -g pnpm
+WORKDIR /app
+COPY package.json pnpm-lock.yaml ./
+RUN pnpm install --no-cache --no-optional && pnpm store prune
```

Comment:
@nabarunchakma da, do you suggest anything to reduce docker image size? In Docker, separate RUN commands create separate layers which increase the docker image size, so better to run command in single line, right?

Reply:
> @nabarunchakma da, do you suggest anything to reduce docker image size? In Docker, if we run separate RUN commands  then, it creates separate layers which increase the docker image size, so better to run command in single line, right?